### PR TITLE
added ESPN

### PR DESCRIPTION
**Pull request by stedy**

### CODE REVIEW

File: sites.json [hunk 1]
```diff
@@ -1857,6 +1857,16 @@
     },
 
     {
+        "name": "ESPN",
+        "url": "http://www.espn.com",
```
Comment (Nightfirecat) [inline sites.json]: Can this become the link to the account settings page?

Comment (stedy) [reply sites.json]: Site uses some sort of Javascript popup window and the base URL remains the same at www.espn.com

Comment (Nightfirecat) [reply sites.json]: Yuck, ok then. Nothing we can do there, I guess.

**Comment by Nightfirecat** [comment]
Thank you for your contribution!

**Pull request merged**